Cargo.toml

```diff
@@ -23,21 +23,19 @@ panic = 'abort'
 
 [dependencies]
 async-dup = "1.2.1"
-async-tls = "0.8.0"
+async-tls = { path = "../async-tls", default-features = false, features = ["client"] }
```

Comment:
Why using a local copy?

Reply:
Not anymore, that was from a previous commit. I had to temporarily use my local patched version of `async-tls` until a new version of it was published today. :)